lib/util/git/index.ts

```diff
@@ -1158,21 +1158,24 @@ export async function pushCommit({
   sourceRef,
   targetRef,
   files,
+  pushOptions,
 }: PushFilesConfig): Promise<boolean> {
   await syncGit();
   logger.debug(`Pushing refSpec ${sourceRef}:${targetRef ?? sourceRef}`);
   let result = false;
   try {
-    const pushOptions: TaskOptions = {
-      '--force-with-lease': null,
-      '-u': null,
-    };
+    const extraArgs = ['--force-with-lease', '-u'];
```

Comment:
Why was this refactoring necessary?

Reply:
That was the only way I found to pass multiple of the same options (various `--options`).

Reply:
`TaskOptions` type only supports `string: null | string | number`. I would need it to support ~` string[]` for:

```js
options: ["some=option", "another=option"]
```

And since that is not supported by simple-git, this was the only alternative approach.

Reply:
@viceice any thoughts?

Reply:
we should wait for the change in simple-git or do this refactor in a separate pr to merge before this pr

Reply:
I personally don't think a new PR is worth, given it's so small and also temporary. I will refactor again anyway as soon as simple-git releases a new version.

But you decide, @viceice, just say the word.